### ISSUE TITLE
index.html not generated for index.scala at the root of test source directory

### ISSUE THREAD

**Issue by shell-software** [etorreborre/specs2#467]
Hi

This is rather a question than an issue

Under **src/test/scala** I have a package **com.example**. In this package I have a file **SampleSpec.scala**. 
I wanted an **index.html** report to be generated, so I created **index.scala** in the root of **src/test/scala** (without any package)

When I try to run tests and generate html report, only **com.example.SampleSpec.html** report is generated.
In case when I move **index.class** into **com.example** package the report is generated successfully, but with fully qualified name: **com.example.index.html**.

I'm using files runner:
```cmd
java -cp <CLASSPATH> org.specs2.runner.files filesrunner.pattern (.*Spec)|index html !pandoc
```

Wondering how **index.html** report can be generated?

**Comment by etorreborre** [comment]
Sorry this is indeed a bug. It is fixed now in `3.7.2-20160329210905-a3be17f` (should be available shortly).

**Comment by shell-software** [comment]
Thank you
